src/form/__tests__/form.test.tsx

```diff
@@ -4,54 +4,63 @@ import React from 'react';
 import { render } from '@testing-library/react';
 import createWrapper from '../../../lib/components/test-utils/dom';
 import Form, { FormProps } from '../../../lib/components/form';
+import liveRegionStyles from '../../../lib/components/internal/components/live-region/styles.selectors.js';
 
 function renderForm(props: FormProps = {}) {
-  const { container } = render(<Form {...props} />);
-  return createWrapper(container).findForm()!;
+  const { container } = render(<Form {...props} errorIconAriaLabel="Error icon" />);
+  return { wrapper: createWrapper(container).findForm()!, container };
 }
 
 describe('Form Component', () => {
   describe('structure', () => {
     it('has no header container when no header is set', () => {
-      const wrapper = renderForm();
+      const { wrapper } = renderForm();
       expect(wrapper.findHeader()).toBeNull();
     });
 
     it('displays header - custom html', () => {
-      const wrapper = renderForm({ header: <h1>Form header</h1> });
+      const { wrapper } = renderForm({ header: <h1>Form header</h1> });
       expect(wrapper.findHeader()!.getElement()).toHaveTextContent('Form header');
     });
 
     it('has no content container when no content is set', () => {
-      const wrapper = renderForm();
+      const { wrapper } = renderForm();
       expect(wrapper.findContent()).toBeNull();
     });
 
     it('displays content - custom html', () => {
-      const wrapper = renderForm({ children: <span>Form content</span> });
+      const { wrapper } = renderForm({ children: <span>Form content</span> });
       expect(wrapper.findContent()!.getElement()).toHaveTextContent('Form content');
     });
 
     it('form contains no error by default', () => {
-      const wrapper = renderForm();
+      const { wrapper } = renderForm();
       expect(wrapper.findError()).toBeNull();
     });
 
     it('form displays the error when set, removes when unset', () => {
-      let wrapper = renderForm({ errorText: 'Some error' });
+      let { wrapper } = renderForm({ errorText: 'Some error' });
       expect(wrapper.findError()!.getElement()).toHaveTextContent('Some error');
 
-      wrapper = renderForm({ errorText: '' });
+      ({ wrapper } = renderForm({ errorText: '' }));
       expect(wrapper.findError()).toBeNull();
     });
 
+    it('form error includes alert and live region', () => {
+      const { container } = renderForm({ errorText: 'Some error' });
+      const wrapper = createWrapper(container);
+
+      expect(wrapper.findAlert()!.getElement()).toHaveTextContent('Some error');
```

Comment:
Why is this not `formWrapper.findError()`?

Reply:
The `findError` finds the error text which is inside the alert, see: https://github.com/cloudscape-design/components/blob/main/src/form/internal.tsx#L33

That was done to not break customers after the error icon has been introduced.

The new test only verifies the presence of the alert and live region and the `findError` selector is used in the above two scenarious.

Reply:
Why not testing it like this

```js
const wrapper = renderForm({ errorText: 'Some error' });

expect(wrapper.findError()!.getElement()).toHaveTextContent('Some error');
// findByClassName works even on `FormWrapper`
expect(wrapper.findByClassName(liveRegionStyles.root)!.getElement()).toHaveTextContent('Error icon, Some error');
```

Reply:
Oh, I didn't consider that -- thanks!